Cargo.toml

```diff
@@ -3,6 +3,10 @@ name = "iggy-py"
 version = "0.2.6"
 edition = "2021"
 authors = ["Dario Lencina Talarico <darioalessandrolencina@gmail.com>"]
+license = "Apache-2.0"
```

Comment:
This won't cross over to pypi but figured that it would be important to use the same copy than the rust client.